doc/user_guide/dynamo0p3.rst

```diff
@@ -421,8 +421,11 @@ types.
 
 1) A Kernel must have at least one argument that is a field, field
    vector, or operator. This rule reflects the fact that a Kernel
-   iterates over a space and therefore must have some representation
-   over that space.
+   operates on some subset of the whole domain (e.g. a cell-column)
+   and is therefore designed to be called from within a loop that
+   iterates over those subsets of the domain. This then only makes
```

Comment:
I think you're right about this sentence just stating the obvious and removing it. Perhaps we will need further explanantion when we have more "iteration domains" with the Physics kernels.

Reply:
Removed :-)